OptimizelySDK/OptimizelyTests/OptimizelyTests-DataModel/ConditionLeafTests.swift

```diff
@@ -35,29 +35,30 @@ class ConditionLeafTests: XCTestCase {
     }
 
     func testDecodeSample_Invalid() {
+        
+        var tmpError: Error?
```

Comment:
Just curious why this new change looks better to you with less readability.

Reply:
it always goes into one of the XCTAssert's, because of which code-cov shows the other two XCTAssert's as uncovered lines. Thats the only reason.